src/creature_states.c

```diff
@@ -456,6 +457,8 @@ struct StateInfo states[CREATURE_STATES_COUNT] = {
     0, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, CrStTyp_AngerJob, 0, 0, 0, 0, 55, 1, 0, 1},
   {creature_going_to_safety_for_toking, NULL, NULL, NULL,
     0, 1, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0,  CrStTyp_Sleep, 0, 0, 1, 0, 54, 1, 0, 1},
+  {creature_timebomb, cleanup_hold_audience, NULL, NULL,
```

Comment:
why does it cleanup hold audience?

Reply:
Because cleanup_hold_audience does exactly what I want, so why make a duplicate function? That said, is it even needed? (Timebombed creatures are doomed to die no matter what)

Reply:
ok, fine, as long as it's no mistake.

Reply:
I see it only corrects the custom speed it has right? Perhaps rename the 'cleanup_hold_audience' function?